### DISCUSSION TITLE
Can someone help me with this?

### DISCUSSION THREAD

**Discussion by mail-main** [nesbox/TIC-80#1673] [General]
I was trying to make a brick breaker game without a tutorial because I thought I understood the api and lua well enough

apparently not.

The sprite I want to put on the screen isn't showing up and the console isn't saying anything. Do keep in mind I am new so this may be incredibly obvious.

Here's the code:

	
	function init()

	bricks={[16]=true,[32]=true,[33]=true}
	p={
	x=200,
	y=200,
	dir=1
	}
	end
	
	init()
	
	function TIC()
	cls()
	map()
	
	spr(256,p.x,p.y) 
	
	end

**Comment by Weeppiko** [comment]
```lua
p={
x=200,
y=200,
dir=1
}
```
You're trying to draw at position 200x200, but TIC resolution is only 240x136, you're just drawing it outside of the screen
